src/screens/Dashboard/DashboardScreen.tsx

```diff
@@ -60,6 +61,12 @@ export class DashboardScreen extends Component<Props, State> {
   walletCarouselRef = React.createRef();
 
   componentDidMount() {
+    RNSecureKeyStore.get('pin')
```

Comment:
would be good to create extra abstraction layer, like `SecureStorageService` or something like that :)
to not directly interact with 'react-native-secure-key-store' in component.

in case we will need to migrate to some other library, it will be also much easier if we have such wrapper.